src/dialog.jsx

```diff
@@ -337,7 +337,7 @@ let Dialog = React.createClass({
       let contentHeight = dialogWindowHeight - dialogContentPadding;
 
       if (this.props.title) contentHeight -= dialogContent.previousSibling.offsetHeight;
-      if (this.props.actions) contentHeight -= dialogContent.nextSibling.offsetHeight;
```

Comment:
@hai, I just did a rebase and pushed; however, I believe someone else fixed the issue with a different commit.

How can we test that this works?